include/revng/ABIAnalyses/UsedReturnValuesOfFunctionCall.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/DenseMap.h"
+#include "llvm/IR/BasicBlock.h"
+#include "llvm/IR/Instruction.h"
+#include "llvm/IR/Value.h"
+
+#include "revng/ABIAnalyses/Common.h"
+#include "revng/BasicAnalyses/GeneratedCodeBasicInfo.h"
+#include "revng/MFP/MFP.h"
+#include "revng/StackAnalysis/StackAnalysis.h"
+#include "revng/Support/revng.h"
+
+namespace UsedReturnValuesOfFunctionCall {
+
+enum State { Unknown, Maybe, Yes };
+
+// special register index, if found inside LatticeElement
+// it means that it is the first time visiting the BasicBlock
+const int32_t InitialRegisterState = -1;
+struct MFI : ABIAnalyses::ABIAnalysis {
+  using LatticeElement = llvm::DenseMap<int32_t, State>;
```

Comment:
Don't use `int32_t`, use a proper `enum` (the one from `Binary.h`, unless there's a good reason not to).

Reply:
Will do as soon as the `feature/stackanalysis` branch is rebased

Reply:
In the end I couldn't use the enums in `Binary.h` as key, I am using the `CSV` variables directly now